From: Markus Koschany <apo@debian.org>
Date: Wed, 23 Aug 2017 02:07:29 +0200
Subject: python-pil compatibility

Python-imaging will be removed from Debian and imports should use python-pil
instead.

Bug-Debian: https://bugs.debian.org/872934
Forwarded: no
---
 src/Texture.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Texture.py b/src/Texture.py
index 9b3b6de..c028d9b 100644
--- a/src/Texture.py
+++ b/src/Texture.py
@@ -24,10 +24,10 @@ from __future__ import division
 
 import Log
 import Config
-import Image
+from PIL import Image
 import pygame
 import StringIO
-import PngImagePlugin
+from PIL import PngImagePlugin
 from OpenGL.GL import *
 from OpenGL.GLU import *
 from Queue import Queue, Empty
